_tools/types/main.go

```diff
@@ -0,0 +1,333 @@
+	fmt.Fprintf(os.Stderr, "%d drivers available on-line\n", len(langs))
+	return drivers, nil
+}
+
+// maybeCloneOrPullAll either clones repos to path in local FS or, if already preset,
+// pulls the latest master for each of them.
+func maybeCloneOrPullAll(drivers []*driverStats) error {
+	fmt.Fprintf(os.Stderr, "cloning %d drivers to %s\n", len(drivers), reposRootPath)
+	err := os.MkdirAll(reposRootPath, os.ModePerm)
+	if err != nil {
+		return err
+	}
+
+	var (
+		cr = make(chan int, 3)
+		wg sync.WaitGroup
+	)
+	for i := range drivers {
+		wg.Add(1)
+		go maybeCloneOrPullAsync(drivers[i], &wg, cr)
+	}
+	wg.Wait()
+	return nil
+}
+
+func maybeCloneOrPullAsync(d *driverStats, wg *sync.WaitGroup, concurent chan int) error {
+	defer wg.Done()
+
+	concurent <- 1
+	defer func() {
+		<-concurent
+	}()
+
+	return maybeCloneOrPull(d)
+}
+
+func maybeCloneOrPull(d *driverStats) error {
+	repoPath := path.Join(reposRootPath, d.path)
+	_, err := os.Stat(repoPath)
+	if err != nil && !os.IsNotExist(err) {
+		fmt.Fprintf(os.Stderr, "error: %s\n", err)
+		return err
+	}
+
+	if os.IsNotExist(err) {
+		fmt.Fprintf(os.Stderr, "%s does not exist, cloning from %s\n", repoPath, d.url)
+		cmd := exec.Command("git", "clone", d.url+".git")
+		cmd.Dir = reposRootPath
+		err = cmd.Run()
+		if err != nil {
+			return err
+		}
+		return nil
+	}
+
+	fmt.Fprintf(os.Stderr, "%s dir exists, will git pull instead\n", repoPath)
+	cmd := exec.Command("git", "pull", "origin", "master")
+	cmd.Dir = repoPath
+	err = cmd.Run()
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+type uastType struct {
+	name string
+}
+
+// isUsedIn checks if current UAST type is used in the given package.
+func (u *uastType) isUsedIn() {
+	// TODO
+}
+
+// findUastTypesInSdk finds all types from SDK.
+func findUastTypesInSdk() []uastType {
+	var out []uastType
+	// TODO: load package, iterate all structs
+	types := []interface{}{
+		uast.Position{},
+		uast.Positions{},
+		// only embeding GenNode?
+		uast.Identifier{},
+		uast.String{},
+		uast.Bool{},
+		uast.QualifiedIdentifier{},
+		uast.Comment{},
+		uast.Group{},
+		uast.FunctionGroup{},
+		uast.Block{},
+		uast.Alias{},
+		uast.Import{},
+		uast.RuntimeImport{},
+		uast.RuntimeReImport{},
+		uast.InlineImport{},
+		uast.Argument{},
+		uast.FunctionType{},
+		uast.Function{},
+	}
+	for _, typee := range types {
+		out = append(out, uastType{reflect.TypeOf(typee).String()})
+	}
+	fmt.Fprintf(os.Stderr, "%d uast:* types found in SDK\n", len(out))
+	return out
+}
+
+// analyzeFixtures goes though all fixtures, assuming the driver is cloned.
+// It updates given driverStats with results.
+func analyzeFixtures(driver *driverStats) error {
+	fixDir := path.Join(reposRootPath, driver.path, fixtureDir)
+	fixtureFiles, err := lsDir(fixDir)
+	if err != nil && !os.IsNotExist(err) {
+		return err
+	}
+	if os.IsNotExist(err) {
+		driver.skip = true
+		return nil
+	}
+	driver.fixturesNum += len(fixtureFiles)
+
+	re := regexp.MustCompile("(uast:[^\"]*)")
```

Comment:
```suggestion
	re := regexp.MustCompile("uast:[^\"]*")
```
since there is only one group, and you are not using `FindSubmatch`